kili/cli/project/export.py

```diff
@@ -21,14 +21,21 @@
     help="Format into which the label data will be converted",
     required=True,
 )
+@click.option(
+    "--output-file", type=str, help="File into which the labels are saved.", required=True
+)
 @click.option(
     "--layout",
     type=click.Choice(get_args(SplitOption)),
     default="merged",
     help="Layout of the label files",
 )
 @click.option(
-    "--output-file", type=str, help="File into which the labels are saved.", required=True
+    "--multiple-files",
```

Comment:
Even if it is not what has been done for the Kili raw format, the default should be multiple files. This is what is done for most of the image annotations formats in Kili. 
So we should set --single-file as a flag, defaulting to False. But I'm interested in you opinion about that!

Reply:
I am ok with that. I think we shoud be consistent with the other exports